string_assignment.js

```diff
@@ -0,0 +1,66 @@
+//STRING EXERCISES
+
+
+// //1.
+// let myStr = 'You can do it!'
+//   console.log(myStr.charAt(0));
+
+
+// //2.
+// let myStr = 'I did it!'
+//   console.log(myStr.length);
+
+// //3.
+// let myStr = 'Oops I did it again'
+//   console.log(myStr.charAt(myStr.length-1));
```

Comment:
Good use of string method here. Although used properly, this code can be simplified by using simple [] notation. Ex: console.log(myStr[myStr.length-1]);